apigw-custom-domain/run.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
that's a good point, and we can include that in the standards going forward, but thinking what we are trying to achieve here, it's to showcase localstack capabilities, not IaC. We have a whole set of samples for different IaC platforms that we can showcase separately. I'd imagine if there was a need to do IaC to a sample, it can be added at later stages, as an alternative makefile entrypoint. 

Also given sometimes limitation/coverage of IaC platform with localstack, AWS CLI is the one that's most likely to work everytime. 

@HarshCasper @robertlcx  what do you think?

Reply:
> showcase localstack capabilities, not IaC

Agree 💯 @bartsdev   Let's adopt what makes sense depending on the context:
* Simple deploy and demonstrator scripts -> AWS CLI is perfectly suitable
* Monolithic bash scripts deploying large complex dependencies -> Adopting IaC is typically easier to understand and reuse compared to maintaining long and brittle bash scripts

> Also given sometimes limitation/coverage of IaC platform with localstack, AWS CLI is the one that's most likely to work everytime.

This used to be the main motivator for the AWS CLI and my point is that we should challenge that default choice where possible. Please report any limitations to the LocalStack Core Team, such they can get fixed. We have an ongoing workstream called [Reduce necessary configuration for deploying AWS stacks on LocalStack](https://www.notion.so/localstack/Reduce-necessary-configuration-for-deploying-AWS-stacks-on-LocalStack-277363db7ba54f2fafa1766b2bae05a1?pvs=4) to highlight the importance of this goal.